src/components/MoneyTransferModal.vue

```diff
@@ -20,11 +20,15 @@
             <p class="control has-icons-left custom-search-bar">
               <input
                 v-model="recipientsSearchString"
-                v-on:input="
-                  recipientsSearchString.length === 0 ||
-                  recipientsSearchString.length >= 3
-                    ? searchRecipients()
-                    : null
+                +
```

Comment:
I think you did not mean to leave this `+` sign here...

Reply:
i forgot it thank you !

Reply:
👍